README.md

```diff
@@ -1,3 +1,4 @@
+![image](https://img.shields.io/github/workflow/status/microsoft/PowerApps-Language-Tooling/CI/master)  ![image](https://img.shields.io/nuget/vpre/Microsoft.PowerPlatform.Formulas.Tools)
```

Comment:
Is there a build status we can include?

Reply:
![image](https://user-images.githubusercontent.com/69215460/110362777-8c739700-7ff6-11eb-9354-d89d1086192b.png)

That first one is the build status, unless I'm misunderstanding

Reply:
looks great.